src/index.js

```diff
@@ -689,24 +676,18 @@ class NearProvider {
      * transaction on the block chain
      * @param {Object} txCallObj transaction call object
      * @property {String} to the address the tx is directed to
-     * @property {String} from (optional) the address the tx is sent
-     * from
-     * @property {Quantity} gas (optional) integer of the gas provided
-     * for the tx execution. `eth_call` consumes zero gas, but this
-     * parameter may be needed by some executions
+     * @property {String} from (optional) the address the tx is sent from
      * @property {Quantity} value (optional) integer of the value sent
      * with this tx
      * @property {String} data (optional) hash of the method signature
      * and encoded parameters
-     * @param {Quantity|Tag} blockHeight integer block number or the
-     * string 'latest', 'earliest', or 'pending'
      * @returns {String} the return value of the executed contract
      */
     async routeEthCall(params) {
-        const {to, value, gas, data} = params[0];
-        let result = await this.account.viewFunction(
-          'evm',
-          'view_call',
+        const {to, /* value, */ data} = params[0];
```

Comment:
`value` is going to be used?

Reply:
to perfectly mimic ETH RPC behavior it needs to be. but the contract can't support it yet